integration/src/test/java/org/mule/test/components/ExtensionWithObjectStoreTestCase.java

```diff
@@ -28,6 +29,9 @@ public class ExtensionWithObjectStoreTestCase extends AbstractIntegrationTestCas
   @Inject
   private ObjectStoreManager objectStoreManager;
 
+  @Inject
+  private org.mule.sdk.api.store.ObjectStoreManager sdkObjectStoreManager;
```

Comment:
For the case of the test done, the connector that injects the SDK-API ObjectStoreManager uses the same sdk-api version as the runtime. Would the injection work if the connector uses a newer or older version of the SDK-API?

Reply:
Not sure if I get your point...the injection would work on both runtime and sdk-api 4.5+